sevntu-checks/src/main/java/com/github/sevntu/checkstyle/checks/coding/MultipleVariableDeclarationsExtendedCheck.java

```diff
@@ -167,9 +167,7 @@ private static DetailAST getLastNode(final DetailAST node) {
         while (child != null) {
             final DetailAST newNode = getLastNode(child);
             if (newNode.getLineNo() > currentNode.getLineNo()
-                    || newNode.getLineNo()
-                        == currentNode.getLineNo() && newNode
```

Comment:
After main repo PR, this line was not covered, and no cases to cover it were discovered in regression testing.